neo/io/igorproio.py

```diff
@@ -101,7 +99,7 @@ def read_analogsignal(self, path=None, lazy=False):
             data = bw.load(self.filename)
             version = data['version']
             if version > 5:
-                raise IOError("Igor binary wave file format version {} "
+                raise OSError("Igor binary wave file format version {} "
```

Comment:
same as previous comment. Keep as IOError for now, to be replaced with a custom exception in a future PR